src/test/java/org/broadinstitute/hellbender/utils/read/markduplicates/MarkDuplicatesSparkTester.java

```diff
@@ -9,6 +10,10 @@
  */
 public final class MarkDuplicatesSparkTester extends AbstractMarkDuplicatesTester {
 
+    public MarkDuplicatesSparkTester() {
+        super(true, DuplicateScoringStrategy.ScoringStrategy.TOTAL_MAPPED_REFERENCE_LENGTH);
```

Comment:
I think we did not before either. See the superclass default constructor - `AbstractMarkDuplicatesTester()` which calls, `SamFileTester` which uses `SAMRecordSetBuilder.DEFAULT_DUPLICATE_SCORING_STRATEGY` which is to `TOTAL_MAPPED_REFERENCE_LENGTH`